shared/containers/AllLayout.js

```diff
@@ -0,0 +1,20 @@
+import MapLayout from './mapLayout';
```

Comment:
ok but why it is not a container but component while it is kinda wrapping all other containers?